vendor/engines/nucore_kfs/lib/nucore_kfs/engine.rb

```diff
@@ -8,9 +8,9 @@
 
 require "nucore_kfs/users_controller_extension"
 
-
 module NucoreKfs
   class Engine < ::Rails::Engine
+    require "nucore_kfs/journal_closer"
```

Comment:
If we are going to inject this method into the controller, the name should change:
```suggestion
    require "nucore_kfs/facility_journals_controller_extension"
```